tests/ltp/patches/ltp_fchmod03_04_fchown03_fix.patch

```diff
@@ -0,0 +1,90 @@
++ Created and deleted temporary directories with root permissions as tests were failing
++ to create directories using nobody.
+diff --git a/testcases/kernel/syscalls/fchmod/fchmod03.c b/testcases/kernel/syscalls/fchmod/fchmod03.c
+index a5824f36b..7330e6234 100644
+--- a/testcases/kernel/syscalls/fchmod/fchmod03.c
++++ b/testcases/kernel/syscalls/fchmod/fchmod03.c
+@@ -141,7 +141,7 @@ void setup(void)
+        tst_sig(NOFORK, DEF_HANDLER, cleanup);
+
+        tst_require_root();
+-
++       tst_tmpdir(); // TODO: Will be removed once fixing git issue :357
+        ltpuser = getpwnam(nobody_uid);
+        if (ltpuser == NULL)
+                tst_brkm(TBROK | TERRNO, NULL, "getpwnam failed");
+@@ -149,7 +149,7 @@ void setup(void)
+
+        TEST_PAUSE;
+
+-       tst_tmpdir();
++//     tst_tmpdir(); // TODO: Will be enabled once fixing git issue :236
+
+        /*
+         * Create a test file under temporary directory with specified
+@@ -165,6 +165,7 @@ void cleanup(void)
+        if (close(fd) == -1)
+                tst_resm(TWARN | TERRNO, "close failed");
+
++       tst_require_root(); // TODO: Will be removed once fixing git issue :357
+        tst_rmdir();
+
+ }
+diff --git a/testcases/kernel/syscalls/fchmod/fchmod04.c b/testcases/kernel/syscalls/fchmod/fchmod04.c
```

Comment:
Please use complete test case name in patch file like... "fix_fchmod03_fchmod04_fchown03.patch". or you can use separate .patch file for each test case.